bindings/pydeck/setup.py

```diff
@@ -208,13 +208,14 @@ def run(self):
         ],
         extras_require={
             "carto": ["pydeck_carto"],
-            "testing": ["pytest"],
+            "jupyter": [
+                "ipywidgets>=7.0.0",
+                'ipython>=5.8.0;python_version<"3.4"',
+                "traitlets>=4.3.2",
+                'ipykernel>=5.1.2;python_version>="3.4"',
```

Comment:
Which of these do we actually depend on? I would've thought we only should require ipywidgets and traitlets, and others would be installed as part of the user's jupyter environment

Reply:
You're correct, I've cut traitlets and ipwidgets. The `.to_html()` function still works.

Reply:
Wait what? I was saying the opposite. We directly depend on `ipywidgets` and `traitlets`... so you certainly need those